takora.spec

```diff
@@ -0,0 +1,34 @@
+Name:           takora
+Version:        0.1.0
+Release:        1%{?dist}
+Summary:        Puppet modules for Ceph installation
+License:        ASL 2.0
+
+URL:            https://github.com/ceph/takora
+
+#Source0:        https://github.com/ceph/%{name}/archive/%{version}.tar.gz
+Source0:       %{name}-%{version}.tar.gz
```

Comment:
I put this part it to make builds work locally for now.  Once we move to Tito and build off of tags, the commented line should work.